net/sslh/src/etc/inc/plugins.inc.d/sslh.inc

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+    Copyright (C) 2022 agh1467@protonmail.com
```

Comment:
```suggestion
    Copyright (C) 2022 NAMEORGITHUBHANDLE <agh1467@protonmail.com>
```

something like this would be nice, ideally we encourage natural names but I can understand not everyone wants that. first name is ok too

Reply:
I'll update to have my github handle.